### PR TITLE
🍰 Feature/new app

### PR DESCRIPTION
**Pull request by exenvee**
## Issue ticket(s) number and link     
- Resolves No link 
     
## Checklist before requesting a review     
- [ ] I have performed a self-review of my code      
- [ ] I have run pre-commit over my code      
- [ ] Code is functional on my machine(Windows 11/Ubuntu22.04 LTS)

### CODE REVIEW

**Review by DmitroPodolsky — CHANGES_REQUESTED** [review]
fix typing

File: app/core/asgi.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/howto/deployment/asgi/
+"""
+
+import os
+
+from django.core.asgi import get_asgi_application
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'core.settings')
```
[the remark this answers is not in the document]
Comment (exenvee) [reply app/core/asgi.py:14]: done

File: app/core/settings.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+# Application definition
+
+INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+]
```
[the remark this answers is not in the document]
Comment (exenvee) [reply app/core/settings.py:40]: done

File: app/core/urls.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+Class-based views
+    1. Add an import:  from other_app.views import Home
+    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
+Including another URLconf
+    1. Import the include() function: from django.urls import include, path
+    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
+"""
+from django.contrib import admin
+from django.urls import path
+
```
[the remark this answers is not in the document]
Comment (exenvee) [reply app/core/urls.py:19]: done

File: app/core/settings.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (RomanShyshcenko) [reply app/core/settings.py:40]: ds

**Review by exenvee — COMMENTED** [review]
Tasks Done

**Review by DmitroPodolsky — APPROVED** [review]
LGTM , merge

**Pull request merged**